packages/nocms-plugin-image-resources/src/createImageResourceProvider.js

```diff
@@ -74,6 +74,12 @@ export default function createImageResourceProvider ({findFiles, readFile, watch
 		}
 	}
 
+	/**
+	 * getImageData returns image meta data
```

Comment:
Lovely with Jsdocs. 

I don't think they recommend repeating the function name here, but just some information of it? It is quiet redundant, after all :) Very small thing.